src/pandablocks_ioc/ioc.py

```diff
@@ -317,6 +317,7 @@ class _RecordUpdater:
     """
```

Comment:
We should add `record_prefix` to the `Args` in the docstring

Reply:
Done